### ISSUE TITLE
Timing out

### ISSUE THREAD

**Issue by astrontelstar** [stackblitz/bolt.new#177]
### Describe the bug

Message did not complete it's response is popping up and I'm having to continually resubmit (burning tokens?)

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-cn8pgb

### Steps to reproduce

Just run the prompt I put in

### Expected behavior

Should complete building the code!

### Screen Recording / Screenshot

<img width="594" alt="image" src="https://github.com/user-attachments/assets/f4fc8a88-c761-4a28-bb16-e559badf2342">

### Platform

```
Browser name = Chrome
Full version = 129.0.0.0
Major version = 129
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 72941454,
  "usedJSHeapSize": 69686498,
  "jsHeapSizeLimit": 4294705152
}
Username = astrontelstar
Chat ID = 2cb753887026
```

**Comment by kirjavascript** [comment]
Hi! Thanks for reporting this issue!

Is this consistent? Can you try again and see if it is still happening? Sometimes messages can get lost for a variety of networking reasons. 

If this is a repeated issue and you still can't continue, we can add this to out internal bug tracker.

**Comment by kc0tlh** [comment]
@astrontelstar resolved in #1073
